### PR TITLE
Add dry bubble

### PR DESCRIPTION
**Pull request by jiahe23**
Add a dry bubble experiment in scampy.

-> Grid mean thetal profile is obtained from LES Dry Bubble experiment and prescribed in 'Case.pyx'
-> Updraft initial thetal and area fraction profiles are obtained from LES Dry Bubble experiment and prescribed in 'EDMF_Updrafts.pyx'
-> parameters are a different set from the other cases

### CODE REVIEW

File: Simulation1d.pyx [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Simulation1d:
         self.Case.initialize_profiles(self.Gr, self.GMV, self.Ref)
         self.Case.initialize_surface(self.Gr, self.Ref )
         self.Case.initialize_forcing(self.Gr, self.Ref, self.GMV)
-        self.Turb.initialize(self.GMV)
+        self.Turb.initialize(self.Case, self.GMV, self.Ref)
```
Comment (ilopezgp) [inline Simulation1d.pyx]: is this needed?

File: Surface.pyx [hunk 2]
```diff
@@ -42,6 +42,26 @@ cdef class SurfaceBase:
...
 
 
+cdef class SurfaceNone(SurfaceBase):
+    def __init__(self):
+        pass
+
+    cpdef initialize(self):
+        return
+    cpdef update(self, GridMeanVariables GMV):
+        # JH: assigning small fluxed so that simulation won't crash when computing mixing length
```
Comment (ilopezgp) [inline Surface.pyx]: I was thinking about this. Do you think the eddy diffusivity should be set to zero in your case in the environment? that is, do you expect a laminar environment with roughly zero environmental TKE? 

If so, we can add conditions that allow a zero TKE to take place.

File: Surface.pyx [hunk 3]
```diff
@@ -42,6 +42,26 @@ cdef class SurfaceBase:
...
+        cdef:
+            Py_ssize_t k, gw = self.Gr.gw
+        self.windspeed = 0.0001
+        self.zrough = 1e-4
+        self.bflux = 1e-4
```
Comment (ilopezgp) [inline Surface.pyx]: We can see at some point how to take care of this to ensure that you can do your experiments with zero fluxes at the surface.

File: Turbulence_PrognosticTKE.pyx [hunk 4]
```diff
@@ -278,8 +278,13 @@ cdef class EDMF_PrognosticTKE(ParameterizationBase):
...
 
-    cpdef initialize(self, GridMeanVariables GMV):
-        self.UpdVar.initialize(GMV)
+    cpdef initialize(self, CasesBase Case, GridMeanVariables GMV, ReferenceState Ref):
+        if Case.casename == 'DryBubble':
+            print 'updraft initialized for Dry Bubble'
+            self.UpdVar.initialize_DryBubble(GMV, Ref)
+        else:
+            print 'updraft initialized for '+Case.casename
+            self.UpdVar.initialize(GMV)
```
Comment (ilopezgp) [inline Turbulence_PrognosticTKE.pyx]: Is there another way to do this? I think that we should be able to make the solver case-independent (keeping the case-dependency to the imposition of initial and boundary conditions)

File: Simulation1d.pyx (hunk 1 above)
Comment (jiahe23) [reply Simulation1d.pyx]: Yes. It calls the following function in Turbulence_PrognosticTKE to initialize the updraft profiles. 

For the bubble case the initial profiles is a perturbed thetali + initial area fraction

The other cases just copy in the grid mean profife.

There are no simple way unless to make it case dependent and reference profile is needed in computing the temperature profiles for the bubble case.

https://github.com/jiahe23/SCAMPy/blob/6b840a9825ec665c8e72c595d3f035e95c621453/Turbulence_PrognosticTKE.pyx#L281

File: Turbulence_PrognosticTKE.pyx (hunk 4 above)
Comment (jiahe23) [reply Turbulence_PrognosticTKE.pyx]: What I could think of is to have a case-dependent initialization file just as in PyCLES? But that seems to much work for now. Maybe we should talk with Charlie make the Julia code capable of doing it?

File: Simulation1d.pyx (hunk 1 above)
Comment (ilopezgp) [reply Simulation1d.pyx]: noted!

File: Turbulence_PrognosticTKE.pyx (hunk 4 above)
Comment (ilopezgp) [reply Turbulence_PrognosticTKE.pyx]: I agree. Can we get rid of the print message for the other cases though? (Maybe keep it just for the bubble)

File: Surface.pyx (hunk 3 above)
Comment (yairchn) [reply Surface.pyx]: take in mind that we are about to move away from scampy after we submit the papers, some large changes I think can wait for the new code where we would take the time to make it more elegant elegant proof

Comment (ilopezgp) [reply Surface.pyx]: ok!

**Pull request merged**